pkg/httpapi/conversion.go

```diff
@@ -24,3 +28,30 @@ func pipelinesToAppsResponse(cfg *config) *appsResponse {
 	}
 	return &appsResponse{Apps: apps}
 }
+
+func applicationsToAppsResponse(appSet []*argoV1aplha1.Application) *appsResponse {
+	appsMap := make(map[string]appResponse)
+	for _, app := range appSet {
+		appName := app.ObjectMeta.Labels["app.kubernetes.io/name"]
```

Comment:
This line is problematic.

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0xb0 pc=0x2170bb6]